### DISCUSSION TITLE
Custom Phrases generated to match.

### DISCUSSION THREAD

**Discussion by inventor7777** [EdgeTX/edgetx#5045] [Q&A] [answered]
Hi!

I would like to add some custom phrases to my MT12 radio.

However, I'm not sure how to use Azure to use the exact same voice to generate these phrases. Is there an easy free way to do this? 

If not, can someone on the team generate them for me? I want to generate these phrases:

"drift mode on"
"drift mode off"
"gyro enabled"
"gyro disabled"
"launch control active"
"wet mode enabled"
"normal mode"
"max gyro"
"max assistance"

**Comment by pfeerick** [comment]
If you think they are phrases other people would want, you can ask here ;) But I will generate those later today sometime. Default English or something else?

https://github.com/EdgeTX/edgetx-sdcard-sounds

**Reply by inventor7777** [reply]
Sorry it took me so long to see your reply @pfeerick! Just standard English to match would be awesome!

**Reply by inventor7777** [reply]
@pfeerick any update on the sounds?
Thanks.

**Reply by pfeerick** [reply, marked as answer]
lol... my turn... sorry for not getting back sooner... edgetx/edgetx-sdcard-sounds#114 adds the initial sounds (they get clipped and normalised on final packaging). Unfortunately it seems like Azure is a bit too clever for it's own good and says gyroscope instead of gyro, so doesn't sound quite as nice as it should.

**Reply by inventor7777** [reply]
@pfeerick thank you very much! The sounds sound great. Sorry for initially opening this here anyway, I should have just done it in the sdsounds repo first.
